src/libsync/propagateupload.h

```diff
@@ -220,5 +259,38 @@ private slots:
     void abortWithError(SyncFileItem::Status status, const QString &error);
 };
 
+/**
+ * @brief The PropagateBundleQNAM class
+ * @ingroup libsync
+ */
+class PropagateBundleQNAM : public PropagateItemJob {
+    Q_OBJECT
+
+private:
+    QVector<AbstractNetworkJob*> _jobs; /// network jobs that are currently in transit
+    QVector<SyncFileItemPtr> _bundledFiles;
+    QJsonObject _bundledFilesMetadata;
+    quint64 _contentID = 1;
```

Comment:
Initializing things here is a C++11 extention that we can't unfortunately use.
(Unless we finaly decide to drop old compilers)
It needs to be initialized in the constructor.